2019q4/foundation.md

```diff
@@ -143,12 +143,12 @@ During the fourth quarter of 2019, Foundation staff continued to improve the
 project's CI infrastructure, worked with contributors to fix the failing build
 and test cases.  We worked with other teams in the project for their testing
 needs and also worked with many external projects and companies to improve
-their support of FreeBSD. We added several new CI jobs and made
+their support of FreeBSD. We added several new CI jobs and brought the
 [FreeBSD Hardware Testing Lab](https://ci.freebsd.org/hwlab) online.
 
 We published
 [2019 in Review: CI and Testing Advancements](https://www.freebsdfoundation.org/blog/2019-in-review-ci-and-testing-advancements/)
-on Foundation's blog.  
+on The Foundation's blog.  
```

Comment:
probably "the Foundation's"?

Reply:
I think you might be right, I think it would be capital The if we said "The FreeBSD Foundation" but since we are just saying "the Foundation," it should be lowercase.  What do you think about changing it to be "The FreeBSD Foundation" instead?

Reply:
@emaste, @brd  could you please decide about "The Foundation"/"the Foundation"/"The FreeBSD Foundation"/"the FreeBSD Foundation"?

Sorry about hurrying, but we are already at the 17th of January and I think we should try to avoid to have the report sitting for such an easy issue.

Reply:
From the Foundation's web site there are examples like "What can I expect from the FreeBSD Foundation?"

So __the Foundation's blog__ or __the FreeBSD Foundation's blog__ as you see fit

Reply:
(sorry I missed this query earlier)

Reply:
Never mind, I see from commits that you are busy with plenty of things :)